Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -18,10 +18,9 @@ public class PlayroomKit
         /// </summary>
         private const string PlayerId = "mockPlayer";
         private static bool mockIsStreamMode;
-        public static Dictionary<string, object> MockDictionary = new();
+        private static Dictionary<string, object> MockDictionary = new();
```

Comment:
Document the reason for this change, in detail, and link the Issue.

Reply:
By document, I simply mean adding a comment which explains why this is private. 😄

Reply:
Added!